src/main/resources/templates/smallfull/approval.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+  <title>Approval</title>
+</head>
+<body>
+<div id="approval-main-content" layout:fragment="content">
+
+  <form id="approval-form" th:action="@{''}" class="form" method="post">
+
+    <h1 id="page-title" class="heading-xlarge">
```

Comment:
Can you use the numberedHeading fragment please? According to the prototype, the approval page should be page number 5.